\title{
A minimal scenario for the origin of non-equilibrium order
}

\author{
Riccardo Ravasio ${ }^{*, 1}$, Kabir Husain, ${ }^{*, 1,2}$, Constantine G. Evans ${ }^{3}$, Rob \\ Phillips $^{4}$, Marco Ribezzi ${ }^{5}$, Jack W. Szostak ${ }^{1}$, Arvind Murugan ${ }^{1}$ \\ ${ }^{1}$ Department of Physics, Chemistry, University of Chicago, \\ Chicago, IL 60637, ${ }^{2}$ Department of Physics and Astronomy, \\ University College London, ${ }^{3}$ Hamilton Institute, \\ Maynooth University, Maynooth, Ireland, ${ }^{4}$ Department of Physics, \\ Caltech, Pasadena, CA 91125, ${ }^{5}$ Laboratoire de Biochimie, \\ Chimie Biologie et Innovation, ESPCI Paris, Université PSL, Paris, France
}

\begin{abstract}
Extant life contains numerous non-equilibrium mechanisms to create order not achievable at equilibrium; it is generally assumed that these mechanisms evolved because the resulting order was sufficiently beneficial to overcome associated costs of time and energy. Here, we identify a broad range of conditions under which non-equilibrium order-creating mechanisms will evolve as an inevitable consequence of self-replication, even if the order is not directly functional. We show that models of polymerases, when expanded to include known stalling effects, can evolve kinetic proofreading through selection for fast replication alone, consistent with data from recent mutational screens. Similarly, replication contingent on fast self-assembly can select for non-equilibrium instabilities and result in more ordered structures without any direct selection for order. We abstract these results into a framework that predicts that self-replication intrinsically amplifies dissipative order-enhancing mechanisms if the distribution of replication times is wide enough. Our work suggests the intriguing possibility that non-equilibrium order can arise more easily than assumed, even before that order is directly functional, with consequences impacting mutation rate evolution and kinetic traps in self-assembly to the origin of life.
\end{abstract}

\section*{I. INTRODUCTION}

Life is often described as a battle against the second law of thermodynamics: ever since Schrödinger's What is life? [1], the ability to maintain a state of unnaturally high order through dissipative mechanisms has been seen as a fundamental characteristic of living matter. Nonequilibrium driving is required to maintain order - i.e., lower entropy or variation than expected at thermal equilibrium - in processes ranging from sensing $[2,3]$, computation $[4-6]$ and spatial organization $[7,8]$ to the selfreplication of heritable information [9-14] itself.

A distinct question from the functioning of nonequilibrium mechanisms is their spontaneous origin; what conditions might drive equilibrium matter to spontaneously start exploiting energy sources in the environment to achieve higher order? The conditions for the emergence of such coupling are not known, even though several ideas have been proposed. One point of view argues that all physical systems have a spontaneous tendency to evolve dissipative structures [15-18]. A more common perspective is that these mechanisms originated through a Darwinian process where the fitness benefits of order were sufficient to overcome the associated costs of energy and time [19-21]; see Fig. 1.

Here, we suggest another possibility, shown in Fig. 1: systems undergoing rapid exponential self-replication can spontaneously evolve non-equilibrium order-enhancing mechanisms if the distribution of replication times is wide enough. In our scenario, order does not need to provide any direct Darwinian fitness benefit; selection can be entirely for faster replication, a selection implicit in any population of self-replicating agents.
We first establish these counter-intuitive ideas in models of proofreading DNA polymerases and then multicomponent self-assembly [22-24]. We expand models in these areas with known biophysical effects such as misincorporation-induced slowing of catalysis during templated replication and kinetic traps in self-assembly. In all cases, we find a counter-intuitive relationship between speed and order - higher speed is associated with higher order because of slowdowns arising from geometric frustration. Consequently, systems with order-enhancing mechanisms like kinetic proofreading $[10,11]$ or dynamic instability [25] replicate faster and will thus be dominant at long times, even if that order is not directly functional in any way.

We then abstract away from these specific instances to determine broad conditions under which non-equilibrium order arises spontaneously; we find that it merely requires self-replication with a sufficiently wide distribution of replication times. Such distributions arise naturally in multi-component systems where replication is contingent on navigating a disordered, high-dimensional configuration space from birth to division. Our results suggest a path through which dissipative order can evolve spontaneously; the order can then become functionalized and consequently selected for, allowing replicating systems to ratchet up in complexity.

\section*{TEMPLATED REPLICATION}

We first explore these ideas in templated replication, a process exploited by all known life forms to transmit heritable information across generations. The fidelity of

Selection for fast replication
![](https://cdn.mathpix.com/cropped/2024_05_22_179932215ae87c71e392g-02.jpg?height=552&width=1742&top_left_y=250&top_left_x=172)

FIG. 1. An alternative scenario for the origin of non-equilibrium order. The basic traits of a self-replicator, such as a cell undergoing repeated divisions as shown, are the time to replicate and fidelity in replication, i.e. lack of disorder; both traits are impacted by dissipative fidelity-enhancing mechanisms. Here, we outline a broad set of conditions where selection for fast replication alone, with no selection on fidelity or order, will result in highly dissipative order-maintaining mechanisms. In our scenario, complex non-equilibrium mechanisms that increase order can arise as an inevitable consequence of exponential self-replication itself, even if there is no fitness benefit to that order. In this scenario, selection for higher fidelity is particularly easy since high fidelity cells will also benefit from higher replication speed. Thus, under the conditions we describe, complex non-equilibrium mechanisms that increase order can arise as an inevitable consequence of exponential self-replication itself, even if there is no fitness benefit to that order.

this replication process is a key physical constraint on the origin of life $[26,27]$ and subsequent evolution $[28]$.

At equilibrium, we expect fidelity to be constrained by the finite differences between base pair free energies; e.g., $\Delta G=G(A T)-G(A G) \approx 1-3 \mathrm{kcal} / \mathrm{mol} \approx 1.688-5 k_{b} T$ would indicate an error rate of $\mu \sim 10^{-1}-10^{-2}[29]$. However, several DNA polymerases contain an exonuclease domain that proofreads $[30,31]$ : the nascent DNA strand is occasionally passed from the polymerase to the exonuclease domain where bases are excised in 3' to 5' direction. Using the exonuclease frequently leads to higher accuracy (lower $\mu$ ) since wrong nucleotides are preferentially discarded; but since right nucleotides are also discarded on occasion, such error correction also increases the time to copy a strand.

A simple mathematical model, following numerous published models [13, 30-33], quantifies this intuitive 'higher fidelity, lower speed' trade-off as shown in Fig. 2a-iii - see Supplementary Information.

We now introduce a key experimentally observed fact - stalling [34-39] - that is not considered in existing models, but completely inverts this intuitive trade-off picture. The incorporation of a wrong nucleotide significantly slows the catalysis of the phosphodiester bond for the next base - even if correct. This biophysical effect of geometric nature [37], due to incorrect seating of the prior incorrect base on the template, e.g. misaligned 3' end, is intrinsic to the nature of templated replication and is independent of proofreading activity: stalling has been observed even in non-enzymatic RNA replication [40] relevant to the RNA world - see Supplementary Information. The time to replicate a strand of length $L$ is,

$$
T_{\text {rep }} \approx L t_{\text {nostall }}(\mu)+L \mu \tau_{\text {stall }}
$$

where $t_{\text {nostall }}(\mu)$ is the time to copy one base, assuming no stalling occurs; this time, shown in Fig.2a-iii, increases as the error rate $\mu$ decreases since additional proofreading activity might excise correctly matched bases as well [10, $11,41,42] ; \mu L$ is the number of mutations, each of which causes a stall of time $\tau_{\text {stall }}$.

Augmenting the proofreading model with stalling reveals a new regime (Fig. 2a-iii): for large enough error rates $\mu \geq \mu_{c}$, the trade-off between time to copy a strand and error is reversed: faster networks are more accurate at replicating the strand. As derived in the Supplementary Information using an analytically tractable model, the width $\mu_{c}$ of this counter-intuitive regime increases with the stalling time $\tau_{\text {stall }}$ and with increasing non-equilibrium drive $\Delta \Phi: \mu_{c} \sim \frac{\tau_{0}}{\tau_{\tau_{\text {stall }}}} f(\Delta \Phi)$, where $\tau_{0}$ is the average incorporation time of a base in the absence of stalling and $f$ is a decreasing function of $\Delta \Phi$. These results suggest that (i) selection for fast replication alone can lead to the evolution of mechanisms that result in higher fidelity; (ii) such higher fidelity will be preferentially achieved through highly dissipative (higher $\Delta \Phi$ ) error-correcting mechanisms.

To test these predictions, we carried out in silico evolution of a DNA polymerase, with fitness agnostic to the copying fidelity $\mu$. Instead, fitness is entirely set by the speed of replicating a strand of length $L, F=1 / T_{\text {rep }}$; this time $T_{\text {rep }}$ includes both proofreading and stalling time as in Eq. 1.
a Stalling inverts speed-accuracy trade-offs

![](https://cdn.mathpix.com/cropped/2024_05_22_179932215ae87c71e392g-03.jpg?height=1208&width=1784&top_left_y=210&top_left_x=170)

(i)

![](https://cdn.mathpix.com/cropped/2024_05_22_179932215ae87c71e392g-03.jpg?height=458&width=1146&top_left_y=430&top_left_x=178)

(ii) (iii)
![](https://cdn.mathpix.com/cropped/2024_05_22_179932215ae87c71e392g-03.jpg?height=640&width=506&top_left_y=211&top_left_x=1406)

Error rate $\mu$

\section*{b In-silico evolution for speed}

(i)

) proofreading cycles

Fitness $=1 /$ Time to replicate

![](https://cdn.mathpix.com/cropped/2024_05_22_179932215ae87c71e392g-03.jpg?height=76&width=412&top_left_y=1328&top_left_x=195)

(iii) Time with no stall $t_{\text {nostall }}$ (iv) Energy dissipation rate (v) Error rate

![](https://cdn.mathpix.com/cropped/2024_05_22_179932215ae87c71e392g-03.jpg?height=385&width=339&top_left_y=1014&top_left_x=1595)

FIG. 2. Fast replication selects for kinetic proofreading in the presence of stalling. (a) We extend canonical models of kinetic proofreading for polymerases (i) by including the stalling effect (ii): incorporation of the correct nucleotide (here, G) at site $i$ is dramatically slowed (up to 1000x [34-36]) if site $i-1$ has an incorrect base (here, red C) due to misaligned 3' of C and 5' end of G. (iii) Average time to copy a strand as a function of error rate in proofreading models without and with stalling. Curves correspond to different non-equilibrium driving potentials $\Delta \Phi$ in the proofreading network. In models with stalling, higher accuracy is linked to higher speed (gray region). (b) (i) In silico evolution of a random network model of polymerases that incorporates stalling. The fitness function is set by the time to copy a long strand, with no consideration of accuracy. (ii,iii,iv,v) Evolution of strand replication time, time to copy a base without a stall, energy dissipation rate and error rate during in silico evolution. Typical trajectory is highlighted out of the ten trajectories reported.

The polymerase was modeled by a generic chemical network composed of $N$ enzyme-substrate states, as sketched in Fig. 2b. We initialize the network's kinetic parameters $k_{i j}$ randomly, but satisfying detailed balance, i.e. at equilibrium. However, $k_{i j}$ are kinetic parameters that can evolve to break detailed balance, representing coupling to external energy sources such as NTP hydrolysis: $k_{i j}=k_{i j}^{\mathrm{eq}} e^{\beta \Delta \Phi}$, where $\Delta \Phi$ is the driving force and $1 / \beta=k_{B} T$ - see Supplementary Information.

The results of the evolution for speed $F=1 / T_{\text {rep }}$ of a network of size $N=7$ are summarized in Fig. 2b. Over cycles of mutation and selection, the replication time $T_{\text {rep }}$ falls as expected. But this fall is accompanied by a decrease in the error rate $\mu$ - despite no selection for fidelity. Further, the time to copy ignoring stalling $t_{\text {nostall }}$, increases during evolution despite selecting for faster replication and energy dissipation also increases, see Fig. 2b.

These observations show that the random network, when selected for fast replication alone, evolved a nonequilibrium error correction mechanism that spends more energy and time (in the sense of higher $t_{\text {nostall }}$ ) proofreading; this increased time $t_{\text {nostall }}$ spent on proofreading each base results in fewer errors and consequently fewer stalling events, leading to a net benefit in replication time $T_{\text {rep }}$ of a strand.

\section*{Experimental evidence}

An extensive body of theoretical - and some experimental - work on proofreading reports the intu-
itive speed-accuracy trade-off. How might our work, motivated by experimental observations of stalling [34$37,39,43]$, be consistent with such prior work? Prior studies consider only the impact of a few mutations or one biophysical perturbation, e.g. changing $\mathrm{Mg}^{2+}$ levels [44]. Such tests cannot probe whether there is a systematic correlation between speed and fidelity. More precisely, trade-offs manifest as a Pareto front defined by an inequality between traits such as speed and accuracy; systems can occupy entire regions to one side of the Pareto front. Upon making a single perturbation to a polymerase, it is possible - and even likely - to find a variant further away from the Pareto front. Comparing a pair of variants, in isolation, can appear to exhibit the intuitive trade-off, but the collection of all mutants can form a Pareto front with the counter-intuitive trade-off. Thus, testing the ideas here presented requires extensive mutagenesis.

![](https://cdn.mathpix.com/cropped/2024_05_22_179932215ae87c71e392g-04.jpg?height=504&width=874&top_left_y=995&top_left_x=170)
- Exo motifs
- Poly motifs

4 Exo + Poly

A WT

FIG. 3. DNA polymerase variants display the counterintuitive trade-off defined as the Pareto front. Error rate vs speed data from the highest throughput mutagenesis of a proofreading DNA polymerase to date [45]. Data points of different shape indicate mutations in the exonuclease, polymerase domain or both. DNA plasmid copy number maintained at steady state by a dedicated DNA polymerase is a measure of its total activity that combines speed and processivity; see Supplementary Information.

The largest such library of a DNA polymerase mutants was built recently [45] in the course of creating the OrthoRep platform [46]. We plotted the data generated by [45] in this process for 171 (single and multiple mutant) variants of the pGKL1 DNA polymerase [47]. This viral-origin Family B DNA polymerase has exonuclease activity and is homologous to the commercially available $\Phi$-29 DNA polymerase. The data in Fig. 2c shows that the counterintuitive trade-off holds over four orders of magnitude variation in the error rate and two orders of magnitude in the measure of a proxy for speed used in [45]. See Supplementary Information for more details and experimental support for the ideas presented here in the origin of life context.

\section*{SELF-ASSEMBLY}

The example of nucleic acid replication is not the only instance of our proposed paradigm. We now consider a cell whose replication is limited by the time to assemble a multicomponent structure; the structure is built from $N$ molecular species whose binding interactions $J_{i j}$ include some non-specific interactions. Extant life must build ordered structures like ribosomes - with every component in the right place despite non-specific interaction - in order to replicate.

In contrast, we will assume that our cell replication is not contingent on the composition or ordering of components in the assembled structure. Our cells divide upon building any structure of linear dimensions at least $L \times W$. Thus, fast replication is equivalent to fast assembly of any structure at all: structural order is not directly functional in our model. This assumption is a theoretical device to highlight the most extreme consequences of the counter-intuitive relationship between replication speed and structural order.

We evolved cells based on such a speed-of-assembly fitness function. We held the interaction matrix $J_{i j}$ fixed but allowed for the possibility of evolving microtubulesinspired dynamic instability [48]; i.e. a layer of nonequilibrium dynamics that allows structures to be partially or entirely disassembled in an irreversible way see Supplementary Information for details.

In silico evolution for fast assembly resulted in faster self-assembly as expected (Fig. 4b), but with two surprises. First, while dynamic instability should naively reduce assembly speed by disassembling structures, the frequency of dynamic instability-disassembly $\lambda$ increased (Fig. 4b-ii) over evolution. Second, despite not selecting for ordered assembly, structural variation in assembled structures at the time of cell division decreased (Fig. 4biii); i.e. assembled structures tended to be ordered in a specific way.

These surprising results can be explained by viewing dynamic instability as a mechanism that speeds up assembly by disassembling misformed structures. As shown in prior work, non-specific interactions can lead to misformed structures [49-52] that dramatically slow further assembly. In the absence of dynamic instability, such misformed leading edges might take a long time to spontaneously melt away. Dynamic instability - of the right amount - might effectively speed up assembly [53] by disassembling these kinetic traps, despite assembly otherwise being strongly forward-driven.

\section*{NAVIGATING HIGH-DIMENSIONAL DISORDERED SPACES}

We abstract our results on kinetic proofreading and self-assembly to a more general framework. Consider a cell that divides upon completing $N$ actions out of a library of $M$ possible actions, in any order and possi-
a Self-assembly with kinetic traps

![](https://cdn.mathpix.com/cropped/2024_05_22_179932215ae87c71e392g-05.jpg?height=1208&width=1764&top_left_y=236&top_left_x=186)

(i) (ii) Correct assembly

\begin{tabular}{|c|c|c|}
\hline ![](https://cdn.mathpix.com/cropped/2024_05_22_179932215ae87c71e392g-05.jpg?height=50\&width=137\&top_left_y=311\&top_left_x=1071) & & ![](https://cdn.mathpix.com/cropped/2024_05_22_179932215ae87c71e392g-05.jpg?height=50\&width=224\&top_left_y=311\&top_left_x=1621) \\
\hline 2| & $\rightarrow$ & ![](https://cdn.mathpix.com/cropped/2024_05_22_179932215ae87c71e392g-05.jpg?height=42\&width=224\&top_left_y=360\&top_left_x=1621) \\
\hline ![](https://cdn.mathpix.com/cropped/2024_05_22_179932215ae87c71e392g-05.jpg?height=43\&width=137\&top_left_y=401\&top_left_x=1075) & Fast & \begin{tabular}{|l|l|l|l|}
3 & 6 & 9 & 12 \\
\end{tabular} \\
\hline
\end{tabular}

(iii)

![](https://cdn.mathpix.com/cropped/2024_05_22_179932215ae87c71e392g-05.jpg?height=179&width=832&top_left_y=482&top_left_x=1042)

(iv)

b In-silico evolution for assembly speed

Evolve: $\lambda$

Fixed: $J_{i j}$

dynamic instability strength: $\lambda$

Fitness $=1 /$ Time [ (v)

![](https://cdn.mathpix.com/cropped/2024_05_22_179932215ae87c71e392g-05.jpg?height=345&width=396&top_left_y=684&top_left_x=1534)

(ii)

![](https://cdn.mathpix.com/cropped/2024_05_22_179932215ae87c71e392g-05.jpg?height=341&width=333&top_left_y=1087&top_left_x=1254)

FIG. 4. Fast replication selects for error-correcting non-equilibrium instabilities in self-assembly. (a) We consider a cell that divides when it assembles any structure of linear size $L=40$ using $N=6$ molecular species; replication is contingent only on structure size and not contingent on ordering of components. The molecules have specific interactions (black entries of $J_{i j}$ ) that enable assembly of a (ii) regular lattice. (iii) But misincorporations due to random non-specific interactions (red entries of $J_{i j}$ ) can lead to frustrated partial assemblies that are slow to grow further. (iv) In our model, cells can potentially evolve a non-equilibrium mechanism that induces frequent irreversible disassembly [48]. (b) In silico evolution with replication being contingent only on assembling structures of length $L$, independent of component ordering. $J_{i j}$ is held fixed but dynamic instability frequency $\lambda$ is allowed to evolve. Even though selection is only for (i) higher assembly speed of any large structure, (ii) dynamic instability strength $\lambda$ and consequently (iii) structural order (measured as variation in component ordering in assembled structures at division) also increase.

bly with repetition. As shown in Fig. 5a-i, such a cell can be seen as navigating one of many possible trajectories $x(t)$ in a high-dimensional configuration space to go from a Birth state to a mature Division state that can then divide into two cells. These actions could represent the order in which different nucleotides are added during templated replication or molecular components are added to, e.g. a growing viral capsid structure [54] or different metabolic and synthesis processes that must be completed during a cell cycle [55].

Equilibrium thermodynamics generally sets a limit on how strictly one can enforce an order of these actions [55], resulting in a relatively wide probability distribution $\mathcal{P}^{e q}[x(t)]$ over trajectories $x(t)$. Extant cells exploit nonequilibrium mechanisms to sample from a narrower dis- tribution $\mathcal{P}^{n e q}[x(t)]$; e.g. typically, only a specific subset of all possible $M$ actions is taken and, in a precise order, to avoid deleterious consequences. This non-equilibrium order can be quantified by the reduction in the entropy of trajectories $\Delta S=D_{K L}\left(\mathcal{P}^{n e q}[x(t)] \| \mathcal{P}[x(t)]\right)$.

Can mechanisms enforcing such non-equilibrium order potentially arise from the need to replicate fast alone? Inspired by the undoing action of proofreading and dynamic instability [20], we consider Maxwell demons (backward-facing blue arrows in Fig. 5a-ii) that randomly and irreversibly reset $[56-58]$ the system back to the Birth state on a timescale $T_{r}$, independent of what state the cell might be at. In contrast with classic Maxwell demons that act based on being in the right or wrong state [55], our mechanism acts solely on time.

\section*{a Division trajectories in high-dimensional state spaces}

![](https://cdn.mathpix.com/cropped/2024_05_22_179932215ae87c71e392g-06.jpg?height=1101&width=1787&top_left_y=230&top_left_x=169)

(i)

![](https://cdn.mathpix.com/cropped/2024_05_22_179932215ae87c71e392g-06.jpg?height=411&width=831&top_left_y=285&top_left_x=189)

(ii)

![](https://cdn.mathpix.com/cropped/2024_05_22_179932215ae87c71e392g-06.jpg?height=463&width=740&top_left_y=243&top_left_x=1170)

b $P\left(T_{\text {rep }}\right)$ : Log-normal distribution

![](https://cdn.mathpix.com/cropped/2024_05_22_179932215ae87c71e392g-06.jpg?height=518&width=509&top_left_y=793&top_left_x=190)

![](https://cdn.mathpix.com/cropped/2024_05_22_179932215ae87c71e392g-06.jpg?height=412&width=401&top_left_y=865&top_left_x=667)

c $P\left(T_{\text {rep }}\right):$ two delta functions

![](https://cdn.mathpix.com/cropped/2024_05_22_179932215ae87c71e392g-06.jpg?height=542&width=364&top_left_y=781&top_left_x=1141)

Replication speed

![](https://cdn.mathpix.com/cropped/2024_05_22_179932215ae87c71e392g-06.jpg?height=502&width=418&top_left_y=790&top_left_x=1531)
(relative fold change)

FIG. 5. Fast replication selects for Maxwell demons that reduce the entropy of birth-to-division paths in high-dimensional state spaces (a) A cell divides when it navigates from a Birth to a Division state along any one of many trajectories $x(t)$ through cell state space. The completion times $\tau$ along these trajectories is distributed as $P(\tau)$. (ii) If $P(\tau)$ satisfies Eq. 2, fast replication selects for a Maxwell demon that randomly and irreversibly resets paths back to Birth on a timescale $T_{r}$. The demon reduces the entropy of paths used to reach Division and thus increases order. (b) (i,ii) Log-normal distributions $P(\tau)$ with a sufficiently large variance $\sigma$ show increased replication speed when reset at a specific timescale (blue region). (iii) The extra order due to the evolved demon, defined as reduction in path entropy, increases with variance $\sigma$ of $P(\tau)$. (c) (i) $P(\tau)$ with two delta functions corresponding to fast $\left(\tau_{f}\right)$ and slow paths $\left(\tau_{s}\right)$; slow paths are taken with probability $\mu$. (ii,iii) Resetting demons are favored for sufficiently large $\tau_{s} / \tau_{f}$ and sufficiently small $\mu$. The solid line in (iii) is the boundary defined by Eq. 2

Prior work $[56,57,59-61]$ suggests that such a reset process can speed up the average transit time from Birth to Division under some conditions. Intuitively, resets reduce time wasted on slow trajectories $x(t)$.

To determine when resetting demons will be selected for by the need for fast self-replication alone - without selecting for order, we first define $P^{e q}\left(T_{\text {rep }}\right)$ to be the distribution of replication times induced by trajectories sampled from the ensemble $P^{e q}[x(t)]$. The fitness $f$ of a lineage of cells with replication time distribution $P^{e q}\left(T_{\text {rep }}\right)$ can be shown to be given by $2 \tilde{P}(-f)=1$ where $\tilde{P}(w)=\int_{0}^{\infty} e^{w T} P^{e q}(T) d T$ is the moment generating function for $P^{e q}\left(T_{r e p}\right)$ - see Supplementary Information and [62]. We can also use similar means to calculate the modified lineage fitness $f\left(T_{r}\right)$ in the presence of a demon that resets on a timescale $T_{r}$ (see Supplementary Information). Setting $f\left(T_{r}\right)>f(\infty)$, we find that speed of replication alone will select a resetting demon of timescale $T_{r}$ if

$$
E(\tau \mid \text { lost })>\frac{T_{r}}{1-P_{\text {lost }}\left(T_{r}\right)}+E(\tau \mid \text { not lost })
$$

where $E(\tau \mid$ lost $)=\int_{T_{r}}^{\infty} T P^{\text {eq }}(T) d t / P_{\text {lost }}\left(T_{r}\right), P_{\text {lost }}\left(T_{r}\right)=$ $\int_{T_{r}}^{\infty} P^{e q}(T) d T$ and $E(\tau \mid$ not lost $)=\int_{0}^{T_{r}} T P^{e q}(T) d T /(1-$ $\left.P_{\text {lost }}\left(T_{r}\right)\right)$. Here 'lost' represents slow trajectories $x(t)$ with long completion times $T_{\text {rep }}>T_{r}$. Intuitively, Eq. 2 amounts to two conflicting requirements on $P^{e q}\left(T_{\text {rep }}\right)$ that must be satisfied for some $T_{r}$ : (i) the average time cost of slow trajectories $E(\tau \mid$ lost) must be large; (ii) the probability of taking these slow trajectories $P_{\text {lost }}\left(T_{r}\right)$ must be small.

While resets are often seen as a way to speed up search [56], the key point in our context is that resetting demons create non-equilibrium order by also modifying $P^{e q}[x(t)]$
to $P^{n e q}[x(t)]$. To see this, note that in the presence of a demon, the Division state can be reached only by a small subset of all possible trajectories $x(t)$, namely the fast ones. Such a reduction in entropy of trajectories is a form of non-equilibrium order. For example, if different trajectories $x(t)$ represent $N$ actions being performed in many different orders, the entropy reduction corresponds to ensuring that those $N$ actions are carried out in a very specific temporal order [55]. Crucially, in many other cases, the trajectories through state space are themselves not observable unless the system successfully reach the Division state; only these successful trajectories that are not reset leave behind an observable fossil record in the form of a molecular structure, e.g. a completed DNA transcript or self-assembled structure not dissolved by dynamic instability. In this context, the reduced entropy of successful trajectories will be directly reflected in, say, lower error rates in newly synthesized DNA strands or higher structural order of macromolecular structures.

The reduction in trajectory entropy $\Delta S$ due to resetting [63] can be bounded in many cases of interest by

$$
\Delta S \leq-\log \int_{0}^{T_{r}} P^{e q}\left(T_{r e p}\right) d T_{r e p}
$$

See Supplementary Information for details. To get intuition on when such order-creating demons will spontaneously evolve and how much order they create, we considered the case of a log-normal distribution for $P^{e q}\left(T_{r e p}\right)$; see Fig. 5b. We find that resets of a specific timescale $T_{r}$ increase fitness by reducing the average time from Birth to Division for log-normal distributions of sufficiently high variance $\sigma$. In this regime, we find effectively higher non-equilibrium order $\Delta S$ since resets greatly restrict the set of trajectories used to reach the Division state. Further, $\Delta S$ increases with variance of $P^{e q}\left(T_{\text {rep }}\right)$ as shown in Fig. 5 b-iii.

As another case study, we consider a bimodal $P^{e q}\left(T_{r e p}\right)$ corresponding to fast trajectories (time $\tau_{f}$ ) with probability $1-\mu$ and slow trajectories (time $\tau_{s}$ ) with probability $\mu$. Such a distribution e.g. models times for templated replication with fast and slow trajectories corresponding to zero and one mutation respectively and where $\mu \ll 1$ is the error rate. Using Eq. 2, we can identify parameter regimes of $\mu, \tau_{s}, \tau_{f}$ in which resets are favorable, $\frac{\tau_{s}}{\tau_{f}}>1+\frac{1}{1-\mu}$ with the resulting order being $\Delta S=-\log (1-\mu)$, due to suppression of mutations. See Fig. 5 c.

In the Supplementary Information, we study other families of replication time distributions $P^{e q}\left(T_{\text {rep }}\right)$, some of which never lead to order-through-speed while others do so for large enough variance, e.g. the normal and Fréchet distributions, respectively.

Real systems can show a distribution of replication times $P^{e q}\left(T_{\text {rep }}\right)$ for numerous mechanistic reasons, ranging from kinetic traps in self-assembly, stalling in DNA replication or other frustrated states along some trajectories $x(t)$ but not others $[62,64,65]$. Independent of mechanism, our results here suggest that these conditions will select for non-equilibrium mechanisms that effectively constrain living systems to show more stereotyped predictable trajectories in going from Birth to Division. Such reliable stereotyped behavior can then be functionalized and directly selected on; but this stereotyping can first arise as a byproduct of fast self-replication.

\section*{PREFERENTIAL EVOLUTION OF DISSIPATIVE ORDER}

a Dissipative proofreading provides more fitness (i)
![](https://cdn.mathpix.com/cropped/2024_05_22_179932215ae87c71e392g-07.jpg?height=344&width=478&top_left_y=770&top_left_x=1084)

(iii)

![](https://cdn.mathpix.com/cropped/2024_05_22_179932215ae87c71e392g-07.jpg?height=347&width=371&top_left_y=778&top_left_x=1579)

b Dissipative instabilities provide more fitness

(i)
![](https://cdn.mathpix.com/cropped/2024_05_22_179932215ae87c71e392g-07.jpg?height=388&width=844&top_left_y=1194&top_left_x=1103)

FIG. 6. Fast replication preferentially selects for more dissipative order-maintaining mechanisms. (a) We compare polymerases (i) restricted to binding state discrimination (linear networks) and (ii) exonuclease-based errorcorrection (networks with loops). Upon selecting for speed, both networks evolve towards lower error rate. But when compared at the same error rate $\mu=0.02$, more dissipative mechanisms have higher replication speed. (b) We compare two mechanisms of error correction in self-assembly: (i) carrying out assembly more reversibly, and (ii) through dynamic instability. Both mechanisms reduce defects while speeding up assembly; but the assembly speed of dynamic instabilitybased dissipative mechanisms is higher while achieve the same defect rate $\mu=0.01$.

Our core argument thus far has been that ordercreating mechanisms may be selected by selection for fast self-replication. However, systems often have a choice of evolving one of many order-maintaining mechanisms that differ in their dissipation cost.

Here, we argue that selection for fast replication alone will preferentially select for order through more dissipative mechanisms over less dissipative mechanisms when
such a choice is available, even when both mechanisms can achieve the same order.

To see this concretely, we consider two families of error correction in templated replication shown in Fig. 6a. While scheme (ii) is a traditional proofreading network with loops, scheme (i) has no discard pathways, but instead is a linear network of states representing a ligandbound state followed by catalysis. The latter network can minimize the formation of errors through slow catalysis and near-equilibrium polymerization [33], a less dissipative way of correcting errors than network (ii).

We separately evolved these two distinct mechanisms for higher speed. While both networks evolved towards lower error rates, the more dissipative system, i.e. network (ii), shows higher fitness, i.e. is faster, even when compared at the same error rate $\mu=0.02$.

We can intuitively understand this effect by computing $\frac{\Delta \mu}{\Delta t_{\text {nostall }} \text {, }}$, i.e., the reduction $\Delta \mu$ in error rates obtained for a given proofreading-associated slow down $\Delta t_{\text {nostall }}$ of copying time in the absence of stalling. As shown in the Supplementary Information, this ratio increases with dissipation.

A similar conclusion holds for self-assembly; besides dynamic instability, self-assembly errors can also be corrected by the less-dissipative mechanism of assembling closer to the structure melting point. However, as shown in Fig. 6b, when achieving the same rate of defects $\mu=0.01$, the dynamic instability mechanism in (ii) is fitter, i.e. assembles structures faster, than the less dissipative mechanism in (i). See Supplementary Information for details.

The above results suggest that the mere fact of exponential proliferation leads to a selection coefficient $s_{\sigma}$ to dissipate more if the distribution of replication times is wide enough. We can estimate $s_{\sigma}$ by adding a dissipation cost $F \rightarrow F-\lambda \sigma$ to the fitness and finding the critical $\lambda_{c}$ that prevents the spontaneous evolution of non-equilibrium order; see Supplementary Information for examples.

\section*{DISCUSSION}

A basic question in the transition from matter to life [9] is to understand the conditions that drive matter to self-organize in a way that increases coupling to external energy sources, thus allowing for more ordered and complex states $[66]$.

Our work outlines one minimal scenario: selfreplication can intrinsically amplify dissipative mechanisms that enhance order if the distribution of replication times has large enough variance. Once such order emerges, it can be functionalized - e.g., accurately copied RNA can begin to code for functional ribozymes, thus adding further selection pressure to maintain that order [27], as in the conventional picture. In this way, our work adds to a line of work on the origin of complexity through ratchets that does not postulate any direct adaptive benefits of complexity $[16,67-70]$.

While competition between extant organisms often involves environment-specific factors beyond fast replication [71], selection for replication speed is the minimal requirement coded in any replicative process. The limiting factor for replication speed can vary, ranging from genome replication (e.g., in some viruses [72]) to assembling macromolecular structures (e.g., viral capsids, ribozymes or ribosomes $[54,73])$ to completing a set of metabolic tasks (e.g., protein synthesis [74]). The three scenarios examined in this paper show that the orderthrough-variance effect is potentially relevant for diverse mechanisms, see Tab. I.

Our work suggests that the route to error-correcting ribozymes [27] in the origin-of-life context might be easier than expected, building on prior stalling-fidelity work [40]. Our work also has implications for speed and accuracy of enzymes of the central dogma [75] such as the ribosome and DNA and RNA polymerases, to the extent they stall upon misincorporations [76]. Our results might also inform efforts to engineer enzymes of higher specificity without loss of speed $[77,78]$. Finally, our counterintuitive speed-accuracy relationship must be accounted for when interpreting mutation rates in natural populations, e.g. our results can explain mutation rates too low to be accounted for by mutational load alone as in the drift-barrier hypothesis $[79]$.

In molecular self-assembly, misincorporation-induced pauses have been studied earlier $[43,49]$ as a way of reducing defects. Our work suggests that such systems with stronger stalled configurations (or with geometric frustration [80]) can more easily evolve non-equilibrium checkpoint mechanisms - e.g. like those during ribosome assembly [81] - even if structural order is not yet directly functional. Similarly, our work adds a twist to annealing protocols studied earlier in crystal and mineral growth [82] which are seen as reducing defects at the cost of growth rate. Our work suggests that, in some regimes, these annealing protocols reduce defects and increase net growth rate at the same time.

Finally, resets have been shown to be a broadly relevant strategy for speeding up search in a broad range of contexts $[56,58,59,61,83]$. Our work points out that in addition to saving time, reset mechanisms effectively reduce the entropy of paths used to reach a destination state. Such 'canalization' into a few paths can be seen as a non-equilibrium version of Waddington's homeorhesis [84]; while Waddington's proposal evoked an equilibrium landscape to canalize the set of allowed trajectories, here, canalization is achieved through non-equilibrium dynamics. While connections to the principle of maximum caliber [85] remain to be explored, we note that the reduction in trajectory entropy can show up as higher observable order in, e.g. assembled structures or copied polymers. As a consequence, complex systems can control themselves and achieve stereotyped reproducible behaviors despite living in high-dimensional disordered state spaces. Taken together, our results suggest a broadly

TABLE I. A minimal scenario for the origin of non-equilibrium order.

Model Key ingredient Non-eq. mechanism Potential relevance

\begin{tabular}{llll} 
Replication (Fig. 2) & Stalling & Kinetic proofreading & \begin{tabular}{l} 
DNA replication, transcription, translation, \\
ribozymes at the origin of life
\end{tabular} \\
Self-assembly (Fig. 4) & Geometric frustration & Dynamic instability & \begin{tabular}{l} 
Viral capsids, microtubules, ribosomal assembly, \\
chaperoned protein folding
\end{tabular} \\
High-dim. trajectories (Fig. 5) & Wide replication times & Random reset & \begin{tabular}{l} 
Cellular replication, cell cycle coordination, \\
check points
\end{tabular}
\end{tabular}

relevant dimensionality reduction principle in correcting errors: instead of developing specific mechanisms that target each of the myriad kinds of errors in a biological process, a simpler alternative strategy is to focus on reducing the time taken by such a process.

\section*{ACKNOWLEDGMENTS.}

We thank John Sutherland, Irene Chen, David Huse, Michael Rust, Erik Winfree, Chang Liu and members of the Murugan group and the CZI theory group for discussions. AM and JWS received support from the Sloan
(G-2022-19518) and Moore (11479) foundations, Matterto-Life program. This work was supported by the NSF Center for Living Systems (grant no. 2317138). AM acknowledges support from NSF PHY-2310781 and NIGMS of the NIH under award number R35GM151211. JWS acknowledges support from NSF grant (2104708). JWS is an Investigator of the Howard Hughes Medical Institute. RP acknowledges the NIH MIRA 1R35 GM11804301. CGE acknowledges support from European Research Council grant no. 772766, Science Foundation Ireland grant nos. 18/ERCS/5746 and 20/FFP-P/8843, and the Evans Foundation for Molecular Medicine. RR and KH acknowledge support from the Yen fellowship.
[1] E. Schrödinger, What is Life? The Physical Aspect of the Living Cell, Canto (Cambridge University Press, Cambridge, 1944).

[2] Y. Tu, Proceedings of the National Academy of Sciences 105,11737 (2008)

[3] P. R. ten Wolde, N. B. Becker, T. E. Ouldridge, and A. Mugler, Journal of Statistical Physics 162, 1395 (2016).

[4] T. E. Ouldridge, C. C. Govern, and P. R. ten Wolde, Physical Review X 7, 021004 (2017).

[5] P. Mehta and D. J. Schwab, Proceedings of the National Academy of Sciences 109, 17978 (2012).

[6] M. E. Raichle and D. A. Gusnard, Proceedings of the National Academy of Sciences 99, 10237 (2002).

[7] T. H. Tan, A. Mietke, J. Li, Y. Chen, H. Higinbotham, P. J. Foster, S. Gokhale, J. Dunkel, and N. Fakhri, Nature 607, 287 (2022).

[8] A. Bacanu, J. F. Pelletier, Y. Jung, and N. Fakhri, Nature Nanotechnology 18, 905 (2023).

[9] J. V. Neumann and A. W. Burks, Theory of SelfReproducing Automata (University of Illinois Press, USA, 1966).

[10] J. Ninio, Biochimie 57, 587 (1975).

[11] J. J. Hopfield, Proceedings of the National Academy of Sciences 71, 4135 (1974).

[12] H. Qian, J. Phys. Chem. B (2006).

[13] A. Murugan, D. A. Huse, and S. Leibler, Physical Review X 4, 021016 (2014).

[14] M. Ehrenberg and C. Blomberg, Biophys. J. 31, 333 $(1980)$.

[15] I. Prigogine and G. Nicolis, in Bifurcation Analysis: Principles, Applications and Synthesis, edited by M. Hazewinkel, R. Jurkovich, and J. H. P. Paelinck (Springer Netherlands, Dordrecht, 1985) pp. 3-12.
[16] T. R. Gingrich, J. M. Horowitz, N. Perunov, and J. L. England, Physical Review Letters 116, 120601 (2016).

[17] J. M. Horowitz, K. Zhou, and J. L. England, Physical Review E 95, 042102 (2017).

[18] J. M. Horowitz and J. L. England, Proceedings of the National Academy of Sciences 114, 7565 (2017).

[19] O. Shoval, H. Sheftel, G. Shinar, Y. Hart, O. Ramote, A. Mayo, E. Dekel, K. Kavanagh, and U. Alon, Science 336, 1157 (2012).

[20] A. Murugan, D. A. Huse, and S. Leibler, Proceedings of the National Academy of Sciences 109, 12034 (2012).

[21] G. Lan, P. Sartori, S. Neumann, V. Sourjik, and Y. Tu, Nature Physics 8, 422 (5012).

[22] M. Nguyen and S. Vaikuntanathan, Proceedings of the National Academy of Sciences 113, 14231 (2016).

[23] G. Bisker and J. L. England, Proceedings of the National Academy of Sciences 115, E10531 (2018).

[24] R. Marsland and J. L. England, Physical Review E 98, 022411 (2018).

[25] T. Mitchison and M. Kirschner, Nature 312, 237 (1984).

[26] G. F. Joyce and J. W. Szostak, Cold Spring Harbor Perspectives in Biology 10, a034801 (2018).

[27] K. F. Tjhung, M. N. Shokhirev, D. P. Horning, and G. F. Joyce, Proceedings of the National Academy of Sciences $117,2906(2020)$.

[28] B. H. Good, M. J. McDonald, J. E. Barrick, R. E. Lenski, and M. M. Desai, Nature 551, 45 (2017).

[29] L. A. Loeb and T. A. Kunkel, Annual Review of Biochemistry 51, 429 (1982).

[30] J. J. Hopfield, Proceedings of the National Academy of Sciences 71, 4135 (1974).

[31] J. Ninio, Biochimie 57, 587 (1975).

[32] B. Munsky, I. Nemenman, and G. Bel, The Journal of Chemical Physics 131, 235103 (2009).

[33] P. Sartori and S. Pigolotti, Physical Review Letters 110, 188101 (2013).

[34] F. W. Perrino and L. A. Loeb, Journal of Biological Chemistry 264, 2898 (1989).

[35] L. V. Mendelman, J. Petruska, and M. F. Goodman, Journal of Biological Chemistry 265, 2338 (1990).

[36] M.-M. Huang, N. Arnheim, and M. F. Goodman, Nucleic Acids Research 20, 4567 (1992).

[37] S. J. Johnson and L. S. Beese, Cell 116, 803 (2004).

[38] J. K. Ichida, A. Horhota, K. Zou, L. W. McLaughlin, and J. W. Szostak, Nucleic Acids Research 33, 5219 (2005).

[39] A. G. Baranovskiy, N. D. Babayeva, A. E. Lisova, L. M. Morstadt, and T. H. Tahirov, Proceedings of the National Academy of Sciences 119, e2111744119 (2022).

[40] S. Rajamani, J. K. Ichida, T. Antal, D. A. Treco, K. Leu, M. A. Nowak, J. W. Szostak, and I. A. Chen, Journal of the American Chemical Society 132, 5880 (2010).

[41] A. A. Johnson and K. A. Johnson, Journal of Biological Chemistry 276, 38097 (2001).

[42] A. Singh, M. Pandey, D. Nandakumar, K. D. Raney, Y. W. Yin, and S. S. Patel, The EMBO Journal 39, e103367 (2020).

[43] C. G. Evans and E. Winfree, in DNA Computing and Molecular Programming, Vol. 11145, edited by D. Doty and H. Dietz (Springer International Publishing, Cham, 2018) pp. 37-54.

[44] M. Johansson, M. Lovmar, and M. Ehrenberg, Current Opinion in Microbiology 11, 141 (2008).

[45] A. Ravikumar, G. A. Arzumanyan, M. K. Obadi, A. A. Javanpour, and C. C. Liu, Cell 175, 1946 (2018).

[46] A. Ravikumar, A. Arrieta, and C. C. Liu, Nature Chemical Biology 10, 175 (2014).

[47] G. Jung, M. C. Leavitt, and J. Ito, Nucleic Acids Research 15, 9088 (1987).

[48] T. Mitchison and M. Kirschner, Nature 312, 237 (1984).

[49] E. Winfree and R. Bekbolatov, in DNA Computing, Lecture Notes in Computer Science, edited by J. Chen and J. Reif (Springer, Berlin, Heidelberg, 2004) pp. 126-144.

[50] E. J. Deeds, O. Ashenberg, J. Gerardin, and E. I. Shakhnovich, Proceedings of the National Academy of Sciences 104, 14952 (2007)

[51] A. Murugan, J. Zou, and M. P. Brenner, Nature Communications 6,6203 (2015).

[52] A. Jhaveri, S. Loggia, Y. Qian, and M. E. Johnson, "Discovering optimal kinetic pathways for self-assembly using automatic differentiation," (2023).

[53] T. E. Holy and S. Leibler, Proceedings of the National Academy of Sciences 91, 5682 (1994).

[54] J. D. Perlmutter and M. F. Hagan, Annual Review of Physical Chemistry 66, 217 (2015).

[55] R. Phillips, The Molecular Switch: Signalling and Allostery (Princeton University Press, 2020).

[56] M. R. Evans, S. N. Majumdar, and G. Schehr, Journal of Physics A: Mathematical and Theoretical 53, 193001 $(2020)$.

[57] M. R. Evans and S. N. Majumdar, Physical Review Letters 106, 160601 (2011).

[58] P. V. Gordon, C. Sample, A. M. Berezhkovskii, C. B. Muratov, and S. Y. Shvartsman, Proceedings of the National Academy of Sciences 108, 6157 (2011).
[59] A. Pal, S. Kostinski, and S. Reuveni, J. Phys. A: Math. Theor. 55, 021001 (2022).

[60] J. Fuchs, S. Goldt, and U. Seifert, Europhysics Letters $113,60009(2016)$.

[61] S. Reuveni, M. Urbakh, and J. Klafter, Proceedings of the National Academy of Sciences 111, 4391 (2014).

[62] F. Jafarpour, C. S. Wright, H. Gudjonson, J. Riebling, E. Dawson, K. Lo, A. Fiebig, S. Crosson, A. R. Dinner, and S. Iyer-Biswas, Physical Review X 8, 021007 (2018).

[63] I. Eliazar and S. Reuveni, J. Phys. A: Math. Theor. 56, $024002(2023)$.

[64] J. Lin and A. Amir, Cell systems 5, 358 (2017).

[65] Y. Lahini, O. Gottesman, A. Amir, and S. M. Rubinstein, Physical Review Letters 118, 085501 (2017).

[66] M. Lynch, Proceedings of the National Academy of Sciences 104, 8597 (2007).

[67] R. E. Lenski, C. Ofria, R. T. Pennock, and C. Adami, Nature 423, 139 (2003).

[68] D. J. Jenkins and D. J. Stekel, Journal of Molecular Evolution 71, 128 (2010).

[69] T. N. Starr, J. M. Flynn, P. Mishra, D. N. A. Bolon, and J. W. Thornton, Proceedings of the National Academy of Sciences 115, 4453 (2018).

[70] A. S. Pillai, S. A. Chandler, Y. Liu, A. V. Signore, C. R. Cortez-Romero, J. L. P. Benesch, A. Laganowsky, J. F. Storz, G. K. A. Hochberg, and J. W. Thornton, Nature 281, $480(2020)$.

[71] M. Basan, T. Honda, D. Christodoulou, M. Hörl, Y.F. Chang, E. Leoncini, A. Mukherjee, H. Okano, B. R. Taylor, J. M. Silverman, C. Sanchez, J. R. Williamson, J. Paulsson, T. Hwa, and U. Sauer, Nature 584, 470 $(2020)$.

[72] P. E. Boehmer and I. R. Lehman, Annu. Rev. Biochem. 66, 347 (1997).

[73] S. Klinge and J. L. Woolford, Nature Reviews Molecular Cell Biology 20, 116 (2019).

[74] M. Scott and T. Hwa, Nat. Rev. Microbiol. 21, 327 $(2023)$.

[75] C. Bustamante, W. Cheng, and Y. X. Mejia, Cell 144, 480 (2011).

[76] C. A. P. Joazeiro, Nat. Rev. Mol. Cell Biol. 20, 368 $(2019)$.

[77] D. S. Tawfik, Current Opinion in Chemical Biology In vivo chemistry $\bullet$ Mechanisms, 21, 73 (2014).

[78] A. I. Flamholz, N. Prywes, U. Moran, D. Davidi, Y. M. Bar-On, L. M. Oltrogge, R. Alves, D. Savage, and R. Milo, Biochemistry 58, 3365 (2019).

[79] M. Lynch, Trends in Genetics 26, 345 (2010).

[80] M. Lenz and T. A. Witten, Nature Physics 13, 1100 $(2017)$.

[81] Z. A. Sanghai, R. Piwowarczyk, A. V. Broeck, and S. Klinge, Nature Structural \& Molecular Biology 30, $594(2023)$.

[82] G. Müller, Cryst. Res. Technol. 42, 1150 (2007).

[83] D. M. Britain, J. P. Town, and O. D. Weiner, eLife 11, e75263 (2022).

[84] C. H. Waddington, Nature 150, 563 (1942).

[85] S. Pressé, K. Ghosh, J. Lee, and K. A. Dill, Rev. Mod. Phys. 85, $1115(2013)$.